doc/how_to/custom_components/reactive_esm/examples/canvas_draw.md

```diff
@@ -0,0 +1,78 @@
+# Build a Custom Canvas Component
```

Comment:
Could we add the examples demoed by Philipp here? At least temporarily. They are a gold mine for me. Without them I have to guess, ask or inspect the underlying implementation to figure out how this is supposed to work.

Reply:
Done, they're in `examples/how_to/custom/`